### PR TITLE
Add coinmarketcap feed customisation

### LINKED ISSUE #3: Add coinmarketcap.com data feed
Add option to select coinmarketcap.com data feed.

https://coinmarketcap.com/api/

https://api.coinmarketcap.com/v1/ticker/lisk/

### PR DESCRIPTION
**Pull request by culshaw**
Fixes #3 

This adds the coinmarketcap feed as a selectable dropdown. 
Along with dynamic currencies from CMC. Future work would be adding a hardcoded list of CMC currencies but I think it would be too many, there's quite a long list.

## CMC feed
screen shot 2018-01-09 at 11 51 02

## LISK explorer feed
screen shot 2018-01-09 at 11 51 10

### CODE REVIEW

File: js/background.js [hunk 1]
```diff
@@ -36,51 +55,78 @@ const receiveData = ({oldValue, newValue}) => {
...
+const parseFeedData = feed => {
+  if (feed) {
+    availableFeeds.forEach(function(item) {
+      if (item.key === feed) {
+        return getData(item.url);
+      }
+    });
+  } else {
+    storage.sync.set({feed: "LSK"});
+    return getData(API.url);
```
Comment (alepop) [inline js/background.js]: `API` is not defined in function scope. This cause an error on the first launch

File: js/background.js [hunk 2]
```diff
@@ -1,7 +1,16 @@
...
+const availableFeeds = [
+  {
+    key: "LSK",
+    url: "https://explorer.lisk.io/api/getPriceTicker"
+  },
+  {
+    key: "CMC",
+    url: "https://api.coinmarketcap.com/v1/ticker/lisk/?convert=EUR"
+  }
+];
```
Comment (alepop) [inline js/background.js]: Better to store sources as a key-value. `{lsk: ... , cmc: ....} `

**Review by alepop — CHANGES_REQUESTED** [review]

**Comment by culshaw** [comment]
@alepop Changes addressed, thanks for reviewing.

**Review by alepop — APPROVED** [review]

**Comment by alepop** [comment]
@culshaw thank you!

**Pull request merged**